core/services/ocr3/plugins/ccip_integration_tests/chainreader/Makefile

```
@@ -0,0 +1,14 @@
+
+SHELL := /opt/homebrew/bin/fish
+
+# IMPORTANT: If you encounter any issues try using solc 0.8.18 and abigen 1.14.5
+
+.PHONY: build
+build:
+	rm -rf build/
+	solc --abi --bin mycontract.sol -o build
```

```suggestion
	solc --evm-version paris --abi --bin mycontract.sol -o build
```

To enforce paris bytecode